### PR TITLE
Run2-TB40 Update TB geometry for June2018 setup

### PR DESCRIPTION
**Pull request by bsunanda**
Correct the version for the June setup

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by davidlange6** [comment]
please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by cvuosalo** [comment]
+1

**Comment by civanch** [comment]
+1

File: Geometry/HGCalCommonData/data/TB181/June18/hgcalBeam.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../../../DetectorDescription/Schema/DDLSchema.xsd">
```
Comment (fabiocos) [inline Geometry/HGCalCommonData/data/TB181/June18/hgcalBeam.xml]: @bsunanda @ianna @cvuosalo this change refers to the fact that there are additional subdirectories in the path defining the location of this file?

Comment (bsunanda) [reply Geometry/HGCalCommonData/data/TB181/June18/hgcalBeam.xml]: Yes it does. Also I missed one for hgcProdcuts.xml which I have now corrected

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by kpedro88** [comment]
+upgrade

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**